### PR TITLE
[-] MO: ganalytics: backoffice is tracking visits!!

### PR DESCRIPTION
**Pull request by kermes**
none of backoffice traffic should be taken into account, all the stats are being broken: visits, bounce rate, time on site....

besides, in backoffice it is using ecommerce.js and ecommerce.js and analytics.js should never be used in the same proprerty:
"Important: The Enhanced Ecommerce plug-in should not be used alongside the Ecommerce (ecommerce.js) plug-in for the same property." (https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce)

### CODE REVIEW

File: ganalytics.php [hunk 1]
```diff
@@ -551,25 +551,25 @@ public function hookAdminOrder()
...
+					if (version_compare(_PS_VERSION_, '1.6', '<') == true)
+						$this->context->controller->addCSS($this->_path.'views/css/ganalytics-nobootstrap.css');
+				}
+				else
+				{
+					$js .= '<link rel="stylesheet" href="'.$this->_path.'views/css/ganalytics.css" type="text/css" />'.
+						'<link rel="stylesheet" href="'.$this->_path.'views/css/ganalytics-nobootstrap.css" type="text/css" />';
+				}
+			}
+
```
Comment (Quetzacoalt91) [inline ganalytics.php]: Nope ! I need this part even if the merchant has not filled the configuration form !
If you move it here, the configuration page will not be designed until the module is configured.

Comment (kermes) [inline ganalytics.php]: All right i just thought was not necessary!

**Comment by kermes** [comment]
Made a wrong merge and it didn't revert as it should. Had to make a bunch of commits to leave all as it was.
@Quetzacoalt91  if it is ok, although right now the code seems fine, just to be sure, I can close this PR and create a new on dev branch with the proper changes.

**Comment by Quetzacoalt91** [comment]
Hi @kermes,

By merging this PR, your whole commit list would be added to the module and there are too many "rollback" in my opinion. I did not have the time yet to check all the content of this PR, but I would like you to create a new one with better commits. :)

Thanks

**Comment by kermes** [comment]
Done, after a thought I made it into 2 PR and made them to dev instead of master.

**Pull request closed without merging**